crates/bevy_math/src/bounding/bounded3d/mod.rs

```diff
@@ -1,6 +1,22 @@
+mod primitive_impls;
+
 use super::BoundingVolume;
 use crate::prelude::{Quat, Vec3};
 
+/// Computes the geometric center of the given set of points.
+#[inline(always)]
+fn point_cloud_3d_center(points: &[Vec3]) -> Vec3 {
+    assert!(
+        !points.is_empty(),
+        "can not compute the center of less than 1 point"
```

Comment:
```suggestion
        "cannot compute the center of an empty set of points"
```
Less than one is strictly zero here, so I think this is clearer